### PR TITLE
final version for prompt lab

### PR DESCRIPTION
**Pull request by krisyifanyang**
This is final version for prompt lab with comments addressed and based on main branch

@maurelya  I addressed your comments. you can look at this one and once this is merged we can closed the old PR.
To try this lab,   go to  dataplatform.cloud.ibm.com/wx

### CODE REVIEW

File: content/Watsonx/WatsonxAI/101.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+[ChatGPT's creators can’t figure out why it won’t talk about Trump](https://www.semafor.com/
+article/02/03/2023/how-chatgpt-inadvertently-learned-to-avoid-talking-about-trump)
+
+```
+
+See: [Example answer](/watsonx/watsonxai/109#2-rewrite)
+
+<p>&nbsp;</p>
+
+### 5.3 Summarize
```
Comment (maurelya) [inline content/Watsonx/WatsonxAI/101.md:290]: For each passage can you add the word `text` after the first ```
This way the format of each passage is more legible similar to the first two passages in the picture below:

Screenshot 2023-09-22 at 10 59 11 AM

**Comment by krisyifanyang** [comment]
Hi @maurelya  I addressed your comments, can you please take a look at it again? Thanks!

**Comment by maurelya** [comment]
Good job! I will approve the PR when Steve has reviewed and approved it.

**Review by ridgill — APPROVED** [review]
I like this for a first pass.  What I think we are missing is the notebook driven approach that Tim used in his bootcamp (https://github.ibm.com/watsonx-ee-squad/workshop-watsonx-ai/tree/main/labs/lab-02-advanced-prompt-engineering).  We can hold off on this until we learn what the watsonx.ai L3 lab will cover because I think this will include quite a bit of prompt lab interaction. 

My other thought is around breakdown/format. Let's consider the exercises and corresponding answers the first actual lab, and the sections 1-4 in 101 current lab should be broken in to a separate page comparable to the "prompt-engineering-basics" in the ai bootcamp lab.  

I'm good merging as it is and addressing these comments in a subsequent PR before we expose the module.

**Pull request merged**